home.md

```diff
@@ -2,33 +2,36 @@
+Archaeogenetics has become a fast accelerating field, with new data coming out faster than many individual researchers can keep track of and co-analyze. Together with samples currently being processed in the world's largest laboratories, we're now approaching genome-wide data for 10,000 ancient individuals. In addition, for many of those samples we also have rich metadata ranging from archaeological information to radiocarbon dating.
 
 The way data is currently shared and published via academic papers, at least from genetic analyses, is mainly via releasing raw sequencing data into public repositories such as the [ENA](https://www.ebi.ac.uk/ena), while providing partial metadata on samples via poorly formatted Excel tables in the Supplement. This creates (at least) the following problems:
 
-1. Intermediate data such as genotypes or metagenomic profiles are often not released at all, making it hard for others to reproduce analyses. 
+1. Intermediate data such as genotypes are often not released at all, making it hard for others to reproduce analyses. 
 2. The connection between individuals, contextual information, and genetic data becomes hard to maintain, bridging between very different repositories and sources (Excel vs. personal homepages vs. public repositories)
-4. Meta-analyses spanning datasets require enormous amounts of work on data collection and curation.
-5. Studies that combine data from different technologies (isotopes, C14 dating, genetics) have no clear way to release such complex relationships in a concise way.
-6. Incrementally produced data, for example by adding new data to  previously published individuals, cannot be easily connected to the same individuals.
+3. Meta-analyses spanning datasets require enormous amounts of work on data collection and curation.
+
+A major initiative to address these problems is the [Allen Ancient DNA Resource](https://reich.hms.harvard.edu/allen-ancient-dna-resource-aadr-downloadable-genotypes-present-day-and-ancient-dna-data) ("AADR"), which is a curated dataset of public ancient DNA data generated, curated and bundled by David Reich's ancient DNA laboratory at Harvard University. In many ways, our initiative is inpiried by and deriving from this resource. In particular, the AADR currently (March 2021) is arguably the most complete resource world-wide that provides genome-wide genotype data for ancient individuals from nearly all publications in the field.
+
+As detailed under [Repositories](repo_guide), our public repository derives to a large extent directly from the AARD, while many curated packages, in particular from 2019 and later, contain data compiled and generated by us. But our initiative also differs in important aspects from the AARD:
```

Comment:
I am okay with this part.

Reply:
from 2019 and later we decided to have data in separate individual packages, we've mostly processed (from BAM files) data from 2020.